components/accelerator/nvidia/error/xid/helper.go

```diff
@@ -83,8 +111,7 @@ func EvolveHealthyState(events []components.Event) (ret components.State) {
 	if lastXidErr == nil {
 		reason = "XIDComponent is healthy"
 	} else {
-		xidErrBytes, _ := lastXidErr.JSON()
-		reason = string(xidErrBytes)
+		reason = fmt.Sprintf("xid %d detected by %s", lastXidErr.Xid, lastXidErr.DataSource)
```

Comment:
@eahydra @cardyok This is where we set the reason that's displayed in the nodes dashboard (prev. the whole JSON structure of

https://github.com/leptonai/gpud/blob/6bc689b15fa8257f61c6156043e0626b4a1ff67e/components/accelerator/nvidia/error/xid/reason.go#L25-L45

now simplified to this one-liner, let me know if you have any suggestions (same to SXid)

Reply:
c.f., https://github.com/leptonai/gpud/issues/391

Reply:
This change has been reverted (same as before) /cc @cardyok